src/components/admin/HeaderButton.tsx

```diff
@@ -1,38 +1,41 @@
 import { Link } from "react-router-dom";
-import classNames from "classnames";
+import cn from "classnames";
+
+type HeaderButtonVariant = "primary" | "secondary" | "multicolor";
+
+const TW = {
+  primary: "text-white bg-sparkle",
+  secondary: "border-gray-300 bg-gray-50 text-gray-700 hover:bg-red-100",
+  multicolor:
+    "text-white bg-gradient-to-r from-sparkle-gradient-blue via-sparkle-gradient-purple to-sparkle-gradient-pink",
+  general:
+    "inline-flex items-center px-4 py-2 border rounded-md shadow-sm text-sm font-medium focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500",
+};
 
 interface HeaderButtonProps {
-  extraClasses?: string;
   name?: string;
   to?: string;
   icon?: (props: React.ComponentProps<"svg">) => JSX.Element;
-  iconExtraClasses?: string;
+  variant?: HeaderButtonVariant;
 }
-export const HeaderButton = ({
-  extraClasses = "border-gray-300 bg-gray-50 text-gray-700 hover:bg-red-100",
+export const HeaderButton: React.FC<HeaderButtonProps> = ({
   to = "#",
   name = "Placeholder",
-  icon = undefined,
-  iconExtraClasses = "text-gray-500",
-}: HeaderButtonProps) => {
   // Alias icon to Icon so that jsx understands it is a component
-  const Icon = icon;
-  return (
-    <Link
-      className={classNames(
-        extraClasses,
-        "inline-flex items-center px-4 py-2 border rounded-md shadow-sm text-sm font-medium focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
-      )}
-      key={name}
-      to={to}
-    >
-      {Icon && (
-        <Icon
-          className={classNames(iconExtraClasses, "-ml-1 mr-2 h-5 w-5")}
-          aria-hidden="true"
-        />
-      )}
-      {name}
-    </Link>
-  );
-};
+  icon: Icon,
+  variant = "primary",
+}: HeaderButtonProps) => (
+  <Link
+    className={cn("HeaderButton", TW.general, TW[variant])}
+    key={name}
+    to={to}
+  >
+    {Icon && (
+      <Icon
+        className={cn("text-gray-500", "-ml-1 mr-2 h-5 w-5")}
```

Comment:
If these aren't going to be put as constants elsewhere, it's simpler to just have 
```suggestion
        className="text-gray-500 -ml-1 mr-2 h-5 w-5"
```

Reply:
yes, you're right, thanks!
https://github.com/sparkletown/sparkle/pull/2812/commits/0f78efe3a381196c9e5b7cacf68aa5c14daa88bc